app/controllers/posts_controller.rb

```diff
@@ -1,6 +1,6 @@
 class PostsController < ApplicationController
   def index
-    @user = User.find_by(id: params[:user_id])
+    @user = User.includes(:posts).find_by(id: params[:user_id])
```

Comment:
- [OPTIONAL] Great job so far 👏🏻 You could also use this approach: `@posts = @user.posts.includes(:comments)`  instead of @posts = @user.posts. For more documentation, visits the [Solving the N+1 problem in Rails](https://medium.com/idopterlabs/solving-the-n-1-problem-in-rails-b5ea677e78e0)